src/MultiDropdown.js

```diff
@@ -3,20 +3,78 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
-
-// HINT User CustomDropdown.js here
-
+import CustomDropdown from "./CustomDropdown";
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = (
+      {
+        "selectedValue1":null,
+        "selectedValue2":null,
+      }
+    );
+  }
+  onChange(selecetedValue,selectedElement) {
```

Comment:
spelling mistake

Reply:
Try splitting this apart into three functions -

1. `onCountryChange`
2. `onCityChange`
3. `onChange`

Reply:
Segmented the onChange function into three different functions and updated the spelling.